### PR TITLE
Add more to introduction

### PR DESCRIPTION
**Pull request by npm1**
This adds a section explaining the largest content and the limitations

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/largest-contentful-paint/pull/40.html" title="Last updated on Aug 28, 2019, 10:27 PM UTC (a1c98b6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/largest-contentful-paint/40/d27588d...a1c98b6.html" title="Last updated on Aug 28, 2019, 10:27 PM UTC (a1c98b6)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -69,7 +69,12 @@ The API relies heavily on [[ELEMENT-TIMING]], which can be thought of as the low
 Elements exposed {#sec-elements-exposed}
 ------------------------
 
-The Largest Contentful Paint API will only expose element types that are already <a>exposed</a> by the Element Timing API.
+The Largest Contentful Paint API will only expose element types that are already <a>exposed</a> by the Element Timing API. In this case, there is no need to annotate them with the <code>elementtiming</code> attribute.
+
+Largest content {#largest-content}
+------------------------
+
+The algorithm used for this API keeps track of the content seen so far. Whenever a new largest content is found, a new entry is created for it. Whenever content is removed, that content is no longer considered by the algorithm. In particular, if the content removed was the largest, then a new entry is created for the new largest. The algorithm terminates whenever scroll or input events occur, since those are likely to introduce new content into the website.
```
Comment (yoavweiss) [inline index.bs]: Is that actually covered by the algorithm? e.g. If I had two pending images in rendering opportunity X, the largest of them gets an entry. If then that image is later removed, how do we know which one is the "now largest" and when it was rendered? Do we keep track of all rendered elements' timestamps?

Comment (npm1) [reply index.bs]: I didn't notice this problem until I added this to the explainer. We do track the 'list of largest', so we need to augment this spec to account for this (keeping an 'ordered list' and updating upon element removal). I filed https://github.com/WICG/largest-contentful-paint/issues/41

Comment (yoavweiss) [reply index.bs]: Thanks for filing #41. Should we block those explainer changes till a fix for it lands?

Comment (npm1) [reply index.bs]: Ok, we can put this on hold for now then.

Comment (npm1) [reply index.bs]: Time to revisit landing this :) WDYT?

**Pull request merged**